qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/QosRateLimiter.java

```diff
@@ -108,8 +111,9 @@ private synchronized void createRateLimiterAtomically() {
         rateLimiter = new SmoothRateLimiter.SmoothBursty(stopwatch, MAX_BURST_SECONDS);
         rateLimiter.setRate(currentRate);
 
-        // TODO(nziebart): distinguish between read/write rate limiters
-        log.info("Units per second set to {}", SafeArg.of("unitsPerSecond", currentRate));
+        log.info("Units per second set to {} for rate limiter type {}",
```

Comment:
(also if we rename stuff, then we should reference it by the new terminology here)